### PR TITLE
Add 2.9 release notes

### PR DESCRIPTION
**Pull request by ohltyler**
### Description
Add 2.9 release notes

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by jackiehanyang — DISMISSED** [review]

File: release-notes/opensearch-anomaly-detection.release-notes-2.9.0.0.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+### Enhancements
+
+* Refactor Shared Settings, Introduce Dynamic Forecast Settings, Rename ([#912](https://github.com/opensearch-project/anomaly-detection/pull/912))
+* Refactoring Index Creation for Improved Code Reuse and Consistency ([#932](https://github.com/opensearch-project/anomaly-detection/pull/932))
+* Add ForecastResult and Refactor Shared Code ([#941](https://github.com/opensearch-project/anomaly-detection/pull/941))
+* Refactor: Migrate files from 'ad' to 'timeseries' package ([#918](https://github.com/opensearch-project/anomaly-detection/pull/918))
+* Add Forecaster class ([#920](https://github.com/opensearch-project/anomaly-detection/pull/920))
+* Refactor: Reorganize code for Forecasting and AnomalyDetector packages ([#925](https://github.com/opensearch-project/anomaly-detection/pull/925))
+* Enforce DOCUMENT Replication for AD Indices and Adjust Primary Shards ([#948](https://github.com/opensearch-project/anomaly-detection/pull/948))
```
Comment (kaituo) [inline release-notes/opensearch-anomaly-detection.release-notes-2.9.0.0.md:11]: There are not in 2.9 release. You can remove them.

File: release-notes/opensearch-anomaly-detection.release-notes-2.9.0.0.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+### Infrastructure
+
+* Updated Maintainers and CODE_OWNERS list ([#926](https://github.com/opensearch-project/anomaly-detection/pull/926))
+* Bump guava version to 32.0.1 ([#933](https://github.com/opensearch-project/anomaly-detection/pull/933))
+* Bump scipy from 1.8.0 to 1.10.0 in /dataGeneration ([#943](https://github.com/opensearch-project/anomaly-detection/pull/943))
+* Fix main build - update import of Releasable and remove reference to BaseExceptionsHelper ([#930](https://github.com/opensearch-project/anomaly-detection/pull/930))
+
+### Documentation
+
+* add 2.8 release notes ([#915](https://github.com/opensearch-project/anomaly-detection/pull/915))
```
Comment (kaituo) [inline release-notes/opensearch-anomaly-detection.release-notes-2.9.0.0.md:22]: This should be in 2.8 release notes. Also, do you need to add current PR to the release note?

File: release-notes/opensearch-anomaly-detection.release-notes-2.9.0.0.md (hunk 1 above)
Comment (kaituo) [reply release-notes/opensearch-anomaly-detection.release-notes-2.9.0.0.md:11]: I meant the following changes are not in 2.9:

* Refactor Shared Settings, Introduce Dynamic Forecast Settings, Rename ([#912](https://github.com/opensearch-project/anomaly-detection/pull/912))
* Refactoring Index Creation for Improved Code Reuse and Consistency ([#932](https://github.com/opensearch-project/anomaly-detection/pull/932))
* Add ForecastResult and Refactor Shared Code ([#941](https://github.com/opensearch-project/anomaly-detection/pull/941))
* Refactor: Migrate files from 'ad' to 'timeseries' package ([#918](https://github.com/opensearch-project/anomaly-detection/pull/918))
* Add Forecaster class ([#920](https://github.com/opensearch-project/anomaly-detection/pull/920))
* Refactor: Reorganize code for Forecasting and AnomalyDetector packages ([#925](https://github.com/opensearch-project/anomaly-detection/pull/925))

File: release-notes/opensearch-anomaly-detection.release-notes-2.9.0.0.md (hunk 2 above)
Comment (ohltyler) [reply release-notes/opensearch-anomaly-detection.release-notes-2.9.0.0.md:22]: Removed. Typically release notes PRs dont need to be in them due to being trivial + chicken/egg problem of adding itself to its own PR.

File: release-notes/opensearch-anomaly-detection.release-notes-2.9.0.0.md (hunk 1 above)
Comment (ohltyler) [reply release-notes/opensearch-anomaly-detection.release-notes-2.9.0.0.md]: Thanks - I've removed

**Review by jackiehanyang — APPROVED** [review]

**Review by kaituo — APPROVED** [review]

**Pull request merged**